examples/example_utils/pubspec.yaml

```diff
@@ -0,0 +1,9 @@
+name: example_utils
```

Comment:
I'm guessing you'll need to add this project to `.github/dependabot.yml` for the next time `flutter_lints` gets updated?

Reply:
Ah thanks for reminding me! I got distracted by fixing diagnostics that I forgot. Done now!